openapi2beans/pkg/cmd/execute.go

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright contributors to the Galasa project
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package cmd
+
+import "github.com/dev-galasa/buildutils/openapi2beans/pkg/utils"
+
+
+func Execute(factory utils.Factory, args []string) error {
+	rootCmd := NewRootCommand(factory, Openapi2beansFlagStore{})
+	rootCmd.SetArgs(args)
+
+	// Execute the command
+	err := rootCmd.Execute()
+
+	return err
```

Comment:
If error is non-nil, you need to set the exit code of the program to 1 rather than 0. Then a calling build process will know if failed, and the build will fail.

Reply:
hmmm, I'm not really sure how to do that

Reply:
have done this in main, so that test code does not hit that